.github/workflows/notification.yml

```diff
@@ -0,0 +1,95 @@
+name: GSI Chatroom PR Notification
+
+on:
+  pull_request:
+    types: [opened, reopened]
```

Comment:
Does it make sense to use `review_requested` here? When a PR opens it is possible not ready yet.

Reply:
Good question and point. I think it's nice to have a notification on `opened` as well. I've added a couple more that I think will be useful. Here is the [full list](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-object-34); take a look and let me know if you think others will be good to add.

Reply:
We call always modify this list if things get to noisy in the channel.